app/controllers/api/ruby_sg_bot_controller.rb

```diff
@@ -0,0 +1,60 @@
+class Api::RubySgBotController < Api::ApplicationController
+  def webhook
+    if join_group?
+      if RubySgBotSubscriber.find_or_create_by(chat_id: tg_chat_id.to_s)
+        telegram_bot_api.send_message(
+          tg_chat_id,
+          <<~TELEGRAM_MESSAGE
+          👋 At your service!
+          TELEGRAM_MESSAGE
+        )
+      end
+    elsif left_group?
+      RubySgBotSubscriber.find_by(chat_id: tg_chat_id.to_s)&.destroy
+    end
+
+    head :no_content
+
+  rescue ArgumentError
+    # if return code is not 2XX, telegram api will keep sending the request.
+    #   There is no error logging for the application for the moment therefore
+    #   just letting it fail silently for now.
+    head :no_content
+  end
+
+  private
+
+  def telegram_bot_api
+    TelegramBotApi.new(ENV["RUBY_SG_BOT_TOKEN"])
+  end
+
+  def rubysg_bot_id
+    "1292813591"
```

Comment:
Can consider using a constant with `.freeze` instead. And since it's a string, `rubysg_bot_id.to_s` in `join_group` and `left_group` might not be necessary?

Reply:
Done